expression/builtin_string_vec.go

```diff
@@ -538,11 +540,37 @@ func (b *builtinConcatWSSig) vecEvalString(input *chunk.Chunk, result *chunk.Col
 }
 
 func (b *builtinConvertSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinConvertSig) vecEvalString(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	expr, err := b.bufAllocator.get(types.ETString, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(expr)
+	if err := b.args[0].VecEvalString(b.ctx, input, expr); err != nil {
+		return err
+	}
+	encoding, _ := charset.Lookup(b.tp.Charset)
+	if encoding == nil {
+		return errUnknownCharacterSet.GenWithStackByArgs(b.tp.Charset)
+	}
+	result.ReserveString(n)
+	result.MergeNulls(expr)
```

Comment:
We can't do that since data stored in `result` is var-length.
PATL: https://github.com/pingcap/tidb/blob/master/util/chunk/column.go#L697

Reply:
ok